tests/suites/test_suite_psa_crypto_driver_wrappers.function

```diff
@@ -1,5 +1,78 @@
 /* BEGIN_HEADER */
 #include "test/drivers/test_driver.h"
+
+#if defined(PSA_WANT_KEY_TYPE_RSA_PUBLIC_KEY)
+/* Sanity checks on the output of RSA encryption.
+ *
+ * \param modulus               Key modulus. Must not have leading zeros.
+ * \param private_exponent      Key private exponent.
+ * \param alg                   An RSA algorithm.
+ * \param input_data            The input plaintext.
+ * \param buf                   The ciphertext produced by the driver.
+ * \param length                Length of \p buf in bytes.
+ */
+static int sanity_check_rsa_encryption_result(
+    psa_algorithm_t alg,
+    const data_t *modulus, const data_t *private_exponent,
+    const data_t *input_data,
+    uint8_t *buf, size_t length )
+{
+#if defined(MBEDTLS_BIGNUM_C)
+    mbedtls_mpi N, D, C, X;
+    mbedtls_mpi_init( &N );
+    mbedtls_mpi_init( &D );
+    mbedtls_mpi_init( &C );
+    mbedtls_mpi_init( &X );
+#endif /* MBEDTLS_BIGNUM_C */
+
+    int ok = 0;
```

Comment:
Maybe `ret` instead of `ok`?

Reply:
We use the convention `ret` for (0 = success, `MBEDTLS_ERR_xxx` = error, sometimes &gt;0 = output size). For boolean return values (1 = success, 0 = failure), we use a different variable name. In the test suites it's typically `ok`.